pkg/compiler/func_scope.go

```diff
@@ -132,6 +132,7 @@ func (c *codegen) countLocalsCall(n ast.Expr, pkg *types.Package) int {
 			switch ce.Args[i].(type) {
 			case *ast.Ident:
 			case *ast.BasicLit:
+			case *ast.SelectorExpr:
```

Comment:
Currently both package variables and struct fields _need_ additional local. We can extend inline implementation to handle package vars similar to local vars, but this won't work with struct fields. Also the test below passes on both this and previous commit and fails if we remove `int` conversion.

Reply:
Cool, I'll fix the test and remove this hunk.

Reply:
Fixed.